configs/components/fesom/fesom-2.1.yaml

```diff
@@ -120,6 +120,7 @@ namelists:
         - namelist.io
         - namelist.cvmix
         - namelist.icepack
+        - namelist.dyn
```

Comment:
This is specific to version 2.5 correct? then it needs to go under the version choose_block

Reply:
One day we could consider (in a discussion with all of you model devs) putting everything into fesom-2.yaml. For now let's keep the minor versions separated.